eclass/xdg-utils.eclass

```diff
@@ -94,8 +95,8 @@ xdg_desktop_database_update() {
 # Updates Gtk+ icon cache files under /usr/share/icons.
 # This function should be called from pkg_postinst and pkg_postrm.
 xdg_icon_cache_update() {
-	has ${EAPI:-0} 0 1 2 && ! use prefix && EROOT="${ROOT}"
-	local updater="${EROOT%/}${GTK_UPDATE_ICON_CACHE}"
+	[[ ${EAPI:-0} == [0123456] ]] && BROOT=${EROOT%/}
```

Comment:
Doesn't this lose early EAPI custom ROOT support, as EROOT doesn't exist there? Albeit in a later commit you drop EAPI-2 and below support in gnome2-utils.eclass at least, sort of. Relying on $PATH would make this moot though, I suppose.

Reply:
Lets make the changes to rely on $PATH then. I've been more or less convinced.